add customization options to quick start (#1942)
meilisearch/documentation: learn/getting_started/quick_start.md

![Meilisearch's search preview indicating the indexes dropdown in the upper right corner](/search_preview/multiple_indexes.png)

## Customization

> At this point, you can configure your Meilisearch instance, and customize your index and search responses.  

Reviewer: Those two `and`s in quick succession bother me a bit. They're not wrong, but perhaps this flows a bit better?

```suggestion
At this point, you can configure your Meilisearch instance and customize your index. When searching, you can use search parameters to refine your results.
```

### Instance options

These allow you to configure your Meilisearch instance at launch with [environment variables](/learn/configuration/instance_options.md#environment-variables) and [command-line options](/learn/configuration/instance_options.md#command-line-options-and-flags). You need to relaunch your instance to alter them.

> Instance options **affect your entire Meilisearch instance**, not just a single index.

Reviewer: Minor, but since the subject is right next to the highlighted sentence, might as well include it and make it a fully-fledged clause.

```suggestion
**Instance options affect your entire Meilisearch instance**, not just a single index.
```

### Index settings

[Index settings](/reference/api/settings.md) allow you to customize search behavior. You can either update all settings globally using the [update settings endpoint](/reference/api/settings.md#update-settings) or individually using a specific child route.

### Search parameters

> [Search parameters](/reference/api/search.md#search-parameters) are used with the search endpoints to improve relevancy. They allow you to alter search responses and behavior.

Reviewer: I _think_ "search results" is more accurate than "search responses"—I'm afraid the second could be understood as the actual response instead of its contents.

```suggestion
[Search parameters](/reference/api/search.md#search-parameters) are used with the search endpoints to improve relevancy. They allow you to alter search results and behavior.
```
